shark/examples/test_cpu.py

```diff
@@ -70,7 +70,7 @@ def test_bert():
     shark_module = SharkInference(
         model, (input,), device="cpu", jit_trace=True
     )
-    results = shark_module.forward((input,))
+    results = shark_module.benchmark_forward((input,))
```

Comment:
In `test_cpu.py` please use forward instead of benchmark_forward since we this will be running in the CI and hence can be slow.